security.md

```diff
@@ -37,3 +37,90 @@ We are working to identify and coordinate security efforts across the Jupyter co
+
+For the purpose of this document these roles are distinct, in practice, some of these roles may be handled by the same individual. However, the roles should be covered by a minimum of two separate individuals. For example, a Reporter may also fill the Developer role and create the remediation, in this case the Coordinator should be a separate individual.
+
+### Process
+
+The role responsible for each step is noted at the beginning.
+
+- Upon receipt of the initial report:
+  - **Coordinator**: Respond to the reported and acknowledge receipt of the report in the timeframe given in the "Reporting Vulnerabilities" section.
+  - **Coordinator**: Open an issue in the private GitHub repository used for tracking vulnerabilities across projects
+  - **Coordinator**: Review the issue for completeness and suitability (triage). If more information is needed, follow up with the Reporter.
+- If the vulnerability is not accepted:
+  - **Coordinator**: Close the issue
+  - **Coordinator**: Notify the reporter
+- If the vulnerability is accepted, within the relevant repositories:
+  - **Coordinator**: Open a draft [GitHub Security Advisory](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#about-github-security-advisories)
+    - Include relevant but sanitized details in the top level comment, which becomes public
+    - Sensitive details and reproductions go in the comments on the draft advisory, which are not public
+    - **Coordinator**: Add relevant people to the advisory - *note* we need an official list of GitHub handles here since teams don't work across orgs
+    - **Developer**: Attempt to replicate the reported vulnerability. Request more information from the **Reporter** if necessary.
+    - **Coordinator**/**Developer**: Request a [CVE](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers) from GitHub. The CVE Number will be private until the advisory is published.
+    - **Developer**: Work on the [vulnerability fix PR](https://docs.github.com/en/code-security/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability#creating-a-temporary-private-fork)
+    - Once the fix is agreed upon, wait for the CVE number to be issued.
+    - **Coordinator**: Notify the **Reporter** of the CVE number. Anyone may mention the CVE number publicly with the affected software name (without revealing version numbers).
+    - **Developer & Coordinator**: Decide on release and announcement dates and post them the draft advisory.  These are typically at least a week apart to allow administrators to deploy fixes.
+  - **Coordinator**: Post the release and announcement dates on the Jupyter Security [Mailing List]([security@ipython.org](mailto:security@ipython.org))
```

Comment:
```suggestion
  - **Coordinator**: Post the release and announcement dates on the [Jupyter Security Mailing List](mailto:security@ipython.org) (`security@ipython.org`)
```

Reply:
there's an incorrectly formatted markdown link in the copied text, I'm not sure if that is also incorrect in the original, but noting it here